packages/vscode-dependencies-validation/README.md

```diff
@@ -0,0 +1 @@
+# vscode-dependencies-validation README
```

Comment:
we need content in this README
We may also want an example demonstrating the functionality.

The README is more urgent